packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -267,9 +271,17 @@ function useModel(model: string | (() => Promise<Model>)): ?Model {
   );
 }
 
+// Override the default mapHitmap with our own implementation.
+const getHitmap: GetHitmap = <T>(prop: T, assignNextIds, alreadySeenObjects) => {
+  const [hitmapProp] = nonInstancedGetHitmap([prop], assignNextIds, alreadySeenObjects);
```

Comment:
Can you just use the non-array version here?
```suggestion
  const hitmapProp = nonInstancedGetHitmap(prop, assignNextIds, alreadySeenObjects);
```